.bashrc

```diff
@@ -0,0 +1 @@
+PYTHONPATH="/home/razor/Projects/alon/meaningful_words_in_txt/"
```

Comment:
why is your bashrc inside this project? OO
this is a pretty personal preference file for your computer.

Reply:
I removed it... haha. 
I was testing some stuff that didn't work for me with Pytest running the tests
I didn't need it there eventually, I deleted it in the next commits

Reply:
be mindful that git saves history +-forever, so every password or secret file that you inserted and deleted in a later commit is compromised unless you took special steps to make sure otherwise.

Reply:
yes, that's just to avoid having the cluttering your history and increasing project size.